src/encoded/static/components/search.js

```diff
@@ -754,6 +754,22 @@ class Facet extends React.Component {
             // The term exists, but without a key, so don't allow its display.'
             return false;
         });
+
+        // Sort numerical terms by value not by frequency
+        // This should ultimately be accomplished in the back end, but the front end fix is much simpler so we are starting with that
+        if (terms.length > 0) {
+            // For straightforward numerical facets, just sort by value
+            // We have to check the full list for now (until schema change) because some lists contain both numerical and string terms ('Encyclopedia version' under Annotations) and we do not want to sort those by value
+            const numericalTest = (a, b) => !isNaN(a) && !isNaN(b);
+            const isNumerical = terms.map(term => term.key).reduce(numericalTest);
+            if (isNumerical === true) {
```

Comment:
Don’t really need to check against `true`. The ESLint rule AirBnB uses is: http://airbnb.io/javascript/#comparison--shortcuts.

Reply:
I originally had the test written without '=== true' but I was finding that isNumerical was returning the term name for facets with only one term name. For instance, "Library depleted in" was returning "rRNA" for isNumerical instead of true or false. Your suggestion of using "every" fixes this problem in addition to being a clearer solution.